app/services/hawkular_proxy_service.rb

```diff
@@ -78,7 +78,7 @@ def data(query)
   rescue StandardError => e
     {
       :parameters => params,
-      :error      => ActionView::Base.full_sanitizer.sanitize(e.message) + " " + _("(Please check your Hawkular server)")
+      :error      => ActionView::Base.full_sanitizer.sanitize(e.to_s) + " " + _("(Please check your Hawkular server)")
```

Comment:
It might also be useful to include the exception class name, e.g. `"#{e.class.name}: #{e}"`.  
Exception messages tend to be written as prodiving extra details for the class, so some are unclear without it.  (The empty message being the worst such case :wink:).